dali_mqtt_daemon.py

```diff
@@ -169,6 +204,83 @@ def initialize_lamps(data_object, client):
         except DALIError as err:
             logger.error("While initializing lamp<%s>: %s", lamp, err)
 
+    for group in groups:
+        logger.debug("Publishing group %d", group)
+        try:
+            logger.debug("Group %s" % group)
+            group_address = address.Group(int(group))
+            
+            actual_level = driver_object.send(gear.QueryActualLevel(group_address))
+            physical_minimum = driver_object.send(
+                gear.QueryPhysicalMinimum(group_address)
+            )
+            min_level = driver_object.send(gear.QueryMinLevel(group_address))
+            max_level = driver_object.send(gear.QueryMaxLevel(group_address))
+            device_name = f"group_{group}"
+
+            group_lamp = device_name
+            logger.debug("Group Name: %s", group_lamp)
+
+            lamp_object = Lamp(
+                log_level,
+                driver_object,
+                device_name,
+                group_address,
+                physical_minimum.value,
+                min_level.value,
+                actual_level.value,
+                max_level.value,
+            )
+
+            data_object["all_lamps"][lamp_object.device_name] = lamp_object
+            group_lamp = lamp_object.device_name
+
+            client.publish(
+                HA_DISCOVERY_PREFIX.format(ha_prefix, group_lamp),
+                lamp_object.gen_ha_config(mqtt_base_topic),
+                retain=True,
+            )
+            client.publish(
+                MQTT_BRIGHTNESS_STATE_TOPIC.format(mqtt_base_topic, group_lamp),
+                actual_level.value,
+                retain=True,
```

Comment:
We should not retain the state... (this and those ahead)

Reply:
This would certainly make sense, but the lamps themselves have also retain=True, should I also change this (Line 165 and following)?

Reply:
Yes, specially if we get that periodic update in ;)

Reply:
Chagned the for retains which change during operation, I thing the other ones are ok like this.